specification/workloads/resource-manager/Microsoft.Workloads/connectors/preview/2023-10-01-preview/connectors.json

```diff
@@ -0,0 +1,1098 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-10-01-preview",
+    "title": "Workloads Client",
+    "description": "The workloads connector client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Workloads/connectors/{connectorName}": {
+      "put": {
+        "tags": [
+          "Connector"
+        ],
+        "operationId": "Connectors_Create",
+        "x-ms-examples": {
+          "Create a connector resource.": {
+            "$ref": "./examples/connectors/Create.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "produces": [
+          "application/json"
+        ],
+        "description": "Creates a connector resource",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
>v3 [](http://example.com/codeflow?start=72&length=2)

In commonTypes.json, you are referring to v5. Standardizing on same version of common types would be better.

Reply:
Microsoft.Workloads/commonTypes.json is common across the Workloads platform where other workloads swaggers also use this apart from our Connector service. This is the reason that there is a difference in actual commonTypes version being referenced.